Auth0/WebAuthError.swift

```diff
@@ -79,6 +80,7 @@ extension WebAuthError {
         switch self.code {
         case .noBundleIdentifier: return "Unable to retrieve the bundle identifier from Bundle.main.bundleIdentifier,"
             + " or it could not be used to build a valid URL."
+        case .transactionActiveAlready: return "Failed to start this transaction, as there is an active transaction at the moment"
```

Comment:
```suggestion
        case .transactionActiveAlready: return "Failed to start this transaction, as there is an active transaction at the moment."
```

Reply:
Updated the text description for the WebAuthError `.transactionActiveAlready` as per description